dapps/onboarding.vue/src/components/accept-contact/accept-contact.ts

```diff
@@ -187,7 +187,7 @@ export default class AcceptContact extends mixins(EvanComponent) {
       }, 2000);
     } catch (ex) {
       dappBrowser.utils.log(ex.message, 'error');
-      (<any> this.$refs.acceptingError).show();
+      (this.$refs.acceptingError as any).show();
```

Comment:
as `EvanModal` ?

Reply:
It's old code. I don't think we should expect a full refactor for everything we touch.